server/service/src/sync/translations/invoice.rs

```diff
@@ -725,7 +739,7 @@ fn to_legacy_confirm_time(
     InvoiceRow {
         r#type,
         picked_datetime,
-        delivered_datetime,
+        received_datetime: delivered_datetime,
```

Comment:
there is only two fields to change, I think should be explicit here

Reply:
Comment still relevant?

Reply:
It does, it's a minor though